lib/bundler/source/git/git_proxy.rb

```diff
@@ -66,7 +66,7 @@ def version
         def checkout
           if path.exist?
             return if has_revision_cached?
-            Bundler.ui.confirm "Updating #{uri}"
+            Bundler.ui.info "Fetching #{uri}"
```

Comment:
I agree that git updates should be info rather than confirm—in the `install` output, confirm means a gem is being installed, while info means a gem is already installed and is simply being used.